operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -15,10 +15,13 @@ Unreleased
 - Add support for adding Annotations to Service (@ofirshtrull)
 - Add `agent.envFrom` value. (@carlosjgp)
 - Add `controller.hostNetwork` value. (@carlosjgp)
+- Move and rename tests to make use of Helm chart-testing. (@carlosjgp)
+- Lint the YAML output of the tests. (@carlosjgp)
```

Comment:
These two aren't user-facing changes to the chart, so typically we'd leave them out of the chagelog (even though we do appreciate the PR).

```suggestion
```